docker-compose.yml

```diff
@@ -38,28 +42,98 @@ services:
     ports:
       - "4100:10000"
 
+  # INSTAGRAM SCRAPER
+  insta_scraper:
+    build:
+      context: "."
+      dockerfile: "insta_scraper/Dockerfile"
+    image: "smag-insta_scraper"
+    depends_on:
+      - "my-kafka"
+    environment:
+      KAFKA_ADDRESS: "my-kafka:9092"
+
+  # TWITTER SCRAPER
+  twitter_scraper-users:
+    build: "twitter_scraper/"
+    image: "smag-twitter_scraper"
+    command: ["-m", "twitterscraper.user_scraper"]
+    depends_on:
+      - "my-kafka"
+    environment:
+      KAFKA_HOST_PORT: "my-kafka:9092"
+      KAFKA_FETCH_TOPIC: "twitter-user_names"
+      KAFKA_INSERT_TOPIC: "twitter-users_scraped"
+  twitter_scraper-tweets:
+    build: "twitter_scraper/"
+    image: "smag-twitter_scraper"
+    command: ["-m", "twitterscraper.tweets_scraper"]
+    depends_on:
+      - "my-kafka"
+    environment:
+      KAFKA_HOST_PORT: "my-kafka:9092"
+      KAFKA_FETCH_TOPIC: "twitter-user_names"
+      KAFKA_INSERT_TOPIC: "twitter-tweets_scaped"
+
+  # TWITTER INSERTER
+  twitter_inserter_users:
+    build:
+      context: "."
+      dockerfile: "./twitter_inserter_users/Dockerfile"
+    image: "smag-twitter_inserter_users"
+    environment:
+      POSTGRES_HOST: "postgres"
+      KAFKA_ADDRESS: "my-kafka:9092"
+      KAFKA_GROUPID: "twitter_inserter_users"
+      KAFKA_INFO_TOPIC: "twitter-users_scraped"
+      KAFKA_NAME_TOPIC: "twitter-user_names"
```

Comment:
I think we agreed to keep the topic names in snake case

Reply:
@Urhengulas

Reply:
We have to refactor the prefix later in order to differentiate better between the different topics.